test-scripts/src/main.rs

```diff
@@ -144,52 +141,27 @@ async fn main() -> anyhow::Result<()> {
         url.to_string()
     };
 
-    let candidates = {
-        if let Some(dir) = args.candidates_dir {
-            let dir = std::fs::read_dir(dir)?;
-            let mut candidates = Vec::new();
-            for entry in dir {
-                let entry = entry?;
-                let path = entry.path();
-                if let Some(ext) = path.extension() {
-                    if ext == "json" {
-                        let mut hasher = sha2::Sha256::new();
-                        let mut file = std::fs::File::open(&path)?;
-                        std::io::copy(&mut file, &mut hasher)?;
-                        let web_path = format!(
-                            "static/concordium/candidates/{}",
-                            path.file_name().context("No file name.")?.to_string_lossy()
-                        );
-                        candidates.push(ChecksumUrl {
-                            url:  make_url(&web_path),
-                            hash: contract::HashSha2256(hasher.finalize().into()),
-                        })
-                    }
-                }
-            }
-            candidates
```

Comment:
This was removed as it is no longer the responsibility of the election http server to serve the candidates data

Reply:
I don't understand this reasoning.

The candidates are still relevant for testing even if they are not served by the election server. You still need them to run an election.

Do you mean that the path has been changed as a result? It's still served from the same "base url" as before.

Reply:
Yes but the reason for supplying them as a directory to the test script was that the election server used to serve them and this would make the testing script more compatible with the election server. The candidate data is not served by the election server however (at least we will not be sure of this; they might host their own stuff?). The election server is no longer configured with a directory holding the candidate data either.